horovod/common/operations.cc

```diff
@@ -1325,6 +1664,11 @@ bool RunLoopOnce(HorovodGlobalState& state, MPIContext& ctx, bool is_coordinator
       LOG(TRACE, state.rank) << "Finished performing " << response.tensor_names_string();
     }
 
+    if (state.response_cache.capacity() > 0) {
+      // Reassign cache bits based on current cache state.
+      state.response_cache.update_cache_bits();
```

Comment:
Should we move this to line 1655, i.e. in prior if statement?

Reply:
Done.